recipe/meta.yaml

```
@@ -204,14 +204,18 @@ outputs:
         - opt_einsum >=2.3.2
         - protobuf >=3.9.2
         - python-flatbuffers >=2
-        - flatbuffers >=2
+        # This can be removed once the dust settles after
+        # the run_export is added to the main and 2.x.x branch
+        # https://github.com/conda-forge/flatbuffers-feedstock/pull/35
+        # https://github.com/conda-forge/flatbuffers-feedstock/pull/36
+        - {{ pin_compatible('flatbuffers', max_pin='x.x') }}
```

```suggestion
        # https://github.com/conda-forge/flatbuffers-feedstock/pull/37
        - {{ pin_compatible('flatbuffers', max_pin='x') }}
```
loosen pin on flatbuffers